jwst/photom/ifu_mrs.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import numpy as np
+from scipy.interpolate import RegularGridInterpolator
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def expfunc_bounded(x, a, b, c, x0):
+    """ Function using time dependent coefficients """
+
+    temp = a * np.exp(-b * (x - x0) / 100) + c
+    temp[temp > 1] = 1
+    return temp
+
+
+def get_correction_function(side, timecoeff, mid_time):
+    """ Find the time and wavelength dependent function """
```

Comment:
needs definition of parameters

Reply:
I agree that having a definition for params like `side` would help a lot.